ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationPeriodValidationIT.scala

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import java.util.regex.Pattern
+
+import com.daml.error.definitions.LedgerApiErrors
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.test.model.Test.Dummy
+import io.grpc.Status
+
+class CommandDeduplicationPeriodValidationIT extends LedgerTestSuite {
+
+  test(
+    "ValidDeduplicationDuration",
+    "Submission returns OK if deduplication time is within the accepted interval",
```

Comment:
This may be enabled for all ledgers, maybe it's worth splitting it off and miving it to a default suite, so that it's always run?

Reply:
This is part of the feature descriptor improvement, so I was thinking of just having it optional for now and in the next PR (which tackles the feature descriptors) run it only for the ledgers that support it